icubam/www/handlers/show.py

```diff
@@ -4,6 +4,7 @@
 from icubam.www.handlers import base
 from icubam.www.handlers import home
 from icubam.www import token
+from icubam.www.handlers.update import time_ago
```

Comment:
Minor comment: i personnally don t like to import functions because it looks like modules and it s more complex to understand when the function come from when looking at the code. Whereas if you do from icubam.www.handlers import update
and then in the code update.time_ago. It is clear where time_ago comes from.

Reply:
please address